src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Linking a SCSS file directly in HTML is incorrect because browsers do not understand SCSS syntax. You should link to a compiled CSS file instead.